dashboard/src/components/StatusBadge.vue

```diff
@@ -3,42 +3,77 @@ import { computed } from "vue";
 
 import type { api } from "@/client";
 
+type badgeType = "package" | "workflow";
 const props = defineProps<{
-  status:
-    | api.EnduroIngestSipStatusEnum
-    | api.EnduroIngestSipWorkflowStatusEnum
-    | api.EnduroIngestSipTaskStatusEnum
-    | api.EnduroStorageAipStatusEnum
-    | api.EnduroStorageAipWorkflowStatusEnum
-    | api.EnduroStorageAipTaskStatusEnum;
+  status: statusEnum;
+  type: badgeType;
   note?: string;
 }>();
 
-const classes: {
-  [key in
-    | api.EnduroIngestSipStatusEnum
-    | api.EnduroIngestSipWorkflowStatusEnum
-    | api.EnduroIngestSipTaskStatusEnum
-    | api.EnduroStorageAipStatusEnum
-    | api.EnduroStorageAipWorkflowStatusEnum
-    | api.EnduroStorageAipTaskStatusEnum]: string;
+type packageEnum =
+  | api.EnduroIngestSipStatusEnum
+  | api.EnduroStorageAipStatusEnum;
+type workflowEnum =
+  | api.EnduroIngestSipWorkflowStatusEnum
+  | api.EnduroStorageAipWorkflowStatusEnum;
+type taskEnum =
+  | api.EnduroIngestSipTaskStatusEnum
+  | api.EnduroStorageAipTaskStatusEnum;
+
+type statusEnum = packageEnum | workflowEnum | taskEnum;
+
+type badgeStyle = string[];
+
+const packageStyle: {
+  [key in packageEnum]: badgeStyle;
 } = {
-  "in progress": "text-bg-info",
-  done: "text-bg-success",
-  error: "text-bg-danger",
-  queued: "text-bg-secondary",
-  pending: "text-bg-warning",
-  canceled: "text-bg-dark",
-  unspecified: "text-bg-dark",
-  stored: "text-bg-success",
-  deleted: "text-bg-danger",
-  processing: "text-bg-info",
-  failed: "text-bg-danger",
-  ingested: "text-bg-success",
+  ingested: [
+    "text-success",
```

Comment:
Is the black text change missing in this PR?

Reply:
Yes, I missed the push somehow :facepalm:  Thanks for catching that @jraddaoui. :)